test/test_transforms_tensor.py

```diff
@@ -725,6 +725,35 @@ def test_autoaugment_save(augmentation, tmpdir):
     s_transform.save(os.path.join(tmpdir, "t_autoaugment.pt"))
 
 
+def test_autoaugment__op_apply_shear():
```

Comment:
LGTM, but shouldn't we add this test to the `test_transforms.py` file instead since this is where the AutoAugment tests live anyway?

Reply:
autoaugment tests live here and there. In `test_transforms.py` we generally test on PIL images and here on tensors. Do you mean that we have to make a similar test on PIL images ? I agree that this can be good as well.